lib/components/__tests__/__snapshots__/tab_view-tests.js.snap

```diff
@@ -6,19 +6,19 @@ exports[`test looks like expected 1`] = `
         Object {
           "alignSelf": "flex-start",
           "borderColor": "red",
-          "borderWidth": 1
+          "borderWidth": 1,
```

Comment:
looks like newer Jest snapshots add dangling commas, probably to make diffs easier to read